app/decorators/lib/decidim/proposals/proposal_serializer_decorator.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Decidim::Proposals::ProposalSerializerDecorator
+  def self.decorate
+    Decidim::Proposals::ProposalSerializer.class_eval do
+      # Add authors to proposal exportation
+      def serialize
```

Comment:
We use to create an alias for the original method and invoke that method from the override and then add the author related fields